backend/core/Http/OpenIdConnectRequestClient.cs

```diff
@@ -95,7 +95,7 @@ public async Task<Models.OpenIdConnectModel> GetOpenIdConnectEndpoints()
         public async Task<string> RequestAccessToken()
         {
             HttpResponseMessage response;
-            if (_accessToken == null || String.IsNullOrWhiteSpace(_accessToken.AccessToken) || _tokenHandler.ReadJwtToken(_accessToken.RefreshToken).ValidTo <= DateTime.UtcNow)
+            if (_accessToken == null || String.IsNullOrWhiteSpace(_accessToken.AccessToken) || (!String.IsNullOrWhiteSpace(_accessToken.RefreshToken) && _tokenHandler.ReadJwtToken(_accessToken.RefreshToken).ValidTo <= DateTime.UtcNow))
```

Comment:
For some reason we are now not always getting a refresh token.  Not sure why, but just need to handle it.